style.css

```diff
@@ -0,0 +1,217 @@
+*{
+    background-color:white
+    
+}
+.container{
+    height: 100vh;
+    display: grid;
+    grid-template-columns: 200px 1fr 300px;
+    margin-right: 80px;
+    margin-left: 80px; 
+    font-family: sans-serif; 
+    gap: 20px;
+}
+
+
+a {
+    text-decoration: none;
+    color: black;
+}
+
+.menu:hover {
+    background-color: rgb(243, 240, 240);
+}
+
+.tweet-feed .home-2:hover{
+    background-color: rgb(243, 240, 240);
+}
+
+
+.tweet-feed .happening-1:hover{
+    background-color: rgb(243, 240, 240);
+}
+
+
+.tweet-feed .reacts-1:hover{
+    background-color: rgb(243, 240, 240);
+}
+
+
+.tweet-feed .reacts-2:hover{
+    background-color: rgb(243, 240, 240);
+}
+
+.side-bar .search:hover{
+    background-color: rgb(243, 240, 240);
+}
+
+.side-bar .happening-2:hover{
+    background-color: rgb(243, 240, 240);
+}
+
+.side-bar .follow:hover{
```

Comment:
Liked all the hovering states. Although they could be nested once all of them use the same declaration of background colour.
`.happening-2:hover, .follow:hover, .reacts-2:hover { bg-color: blabla; }`